### DISCUSSION TITLE
Question about setting networks for containers

### DISCUSSION THREAD

**Discussion by RiffyDivine** [louislam/dockge#560] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I am still new to all this so forgive me. I am trying to learn to play with traefik and prefer using dockge. My issue seems to be when I make the traefik network for containers to attach onto it makes both a network in the name of the container and even with the box ticked for the external traefik network I don't see anything attaching to it besides the traefik container. 

Is there a step I am missing to make it part of the traefik network? I am working off them not being on the same network because traefik isn't seeing it and the ip address of both containers aren't in the same range. This is the first time I've had to try and put all my containers on the same network so I am not sure what step I am missing. The network they are trying to connect into I made before the containers with the docker cli and have it bridged. 

### 📝 Error Message(s) or Log

none.

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Ubuntu 24.04

### 🌐 Browser

Firefox

### 🐋 Docker Version

27.0.3

**Comment by Hypnos-69** [comment, marked as answer]
Hello,

I had the same "problem". Checking off our external network does not mean that the container uses it.
![image](https://github.com/user-attachments/assets/9adce204-cbfe-4216-a240-26ff751a4a9f)

To use it, having "enabled" the use of our external network, you need to edit the stack configuration :
![image](https://github.com/user-attachments/assets/43ef5ab0-b2e3-4196-9d92-505a3e597af7)

And finally we need to configure the external network to be used :
![image](https://github.com/user-attachments/assets/86ecee35-5a3a-4562-bf13-15801e564e12)

Once this is done, my compose.yaml looks like this :
![image](https://github.com/user-attachments/assets/e11af257-f5a7-48ce-acd7-fa79091528f4)

As soon as I had set up my stack like this, my container started up using my "gizmo" external network.

Best regards,

Alex
